ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -43,105 +38,47 @@ import scala.util.{Failure, Success}
 
 /** @param dispatcher Executes the queries prepared by this object
   * @param executionContext Runs transformations on data fetched from the database, including Daml-LF value deserialization
-  * @param pageSize The number of events to fetch at a time the database when serving streaming calls
-  * @param eventProcessingParallelism The parallelism for loading and decoding state events
+  * @param payloadProcessingParallelism The parallelism for loading and decoding event payloads
   * @param lfValueTranslation The delegate in charge of translating serialized Daml-LF values
-  * @see [[PaginatingAsyncStream]]
   */
 private[dao] final class TransactionsReader(
+    flatTransactionsStreamReader: TransactionsFlatStreamReader,
```

Comment:
We have a patchy parameter documentation in this function

Reply:
Fixed